### ISSUE TITLE
Ensuring charters  fit the mission and values of W3C

### ISSUE THREAD

**Issue by michaelchampion** [w3c/w3process#620]
Turning @mnot's  https://github.com/w3c/w3process/issues/328#issuecomment-1210093948 into a standalone issue: In a W3C without an engaged Director, how do we ensure that chartered work is consistent with W3C's mission and values?  

Underlying problem: Initially W3C had a Director to make judgments on how to maintain the conceptual integrity of the w3c's standards system, but now that role is essentially vacant. So we need some mechanism for the community  to evaluate such matters at charter time, and encourage any concerns to be explicitly raised and authoritatively resolved during the approval process.

Symptoms: W3C's recent public controversies tend to happen when a spec gets to Proposed Recommendation and only then are are fundamental questions surfaced about its appropriateness for W3C's mission, culture, implicit values, etc.  The obvious example is EME https://en.wikipedia.org/wiki/Encrypted_Media_Extensions#Criticism .  We can certainly expect future controversies based on privacy and sustainability concerns.  Once W3C is launched as a non-profit corporation without even nominal supervision from academic institutions or Director applying an explicit vision and implicit values to approve work at various maturity stages, it will be even more important to ensure such issues are resolved early.  Rejecting a spec worked on for years for fundamental philosophical reasons will seem like a betrayal of trust and drive members away.  Conversely, not ensuring that W3C's Recommendations align with a coherent mission and set of values will drive away those who engage in W3C hoping to close gaps between the web's "full potential" and current (sometimes ugly) reality. 

Possible solutions:

1. Tweak the current / proposed Director-free process to ensure charters get horizontal review to surface issues and the AC is informed of these results when voting on a charter. That seems to be the intent of #328, but as @mnot notes that is "sweeping some signficant problems under the carpet." 

2. @mnot's suggestion: Make the TAG (or possibly TAG+AB) "responsible for new charters (much as in the IETF)" .  But this will add to the already significant burdens of the TAG, and it's not clear that the current TAG members want the role.

3. My previous suggestion in #457 and embedded is other issues for a Technical Director(ate) that would be explicitly hired/elected/appointed to be responsible for issues at the intersection of architecture and strategy such as the problem here.  But that proposal has gotten little support, and raises the obvious question of who exactly W3C could entrust with such responsibility and how to find a qualified independent person W3C could afford to hire.  Plus this would require engagement of the  Board of Directors who will have their hands full restructuring the current organization.

Perhaps there are other possibilities, or an efficient / workable way of combining some of these options to address the problem.

Also, perhaps this issue must be deferred until there is an authoritative and operational statement of what W3C's mission and values are.  This has been an AB "priority" for some time now, and perhaps will get some cycles once a BoD takes over the job of advising and supervising the team on "business" matters. But I suspect discussions of the *process* for ensuring the values are respected in charters can proceed in parallel with discussions of the *content* of the values.

**Comment by jeffjaffe** [comment]
It requires AC approval to charter new work.  If some stakeholders believe that a charter does not fit the mission and values of W3C, they would object (this has happened in the past).  That objection would be resolved as any other objection (e.g. Council).  It is not obvious that we need anything more process-wise to address this issue.

**Comment by plehegar** [comment]
The guide documents horizontal reviews ofcharters: https://www.w3.org/Guide/process/charter.html

**Comment by dwsinger** [comment]
+1 to Jeff. This is a (major) reason for AC review. Whether the AC can see what comment HR groups, TAG, AB, and (for charters offered by other than the Team) the Team might have, is a separate issue.

**Comment by plehegar** [comment]
(feel free to open new issues that are more specific)

**Comment by michaelchampion** [comment]
So you are comfortable replacing the Director's expert, principled strategic guidance with a political process?  

When I was an AC Rep, I was once told by the team that I had "exceeded my quota of formal objections" by voting against too many charters. AC Reps have NO incentive to run a rigorous charter review by the experts in their company, and are actively discouraged from objecting unless a a charter is so toxic it would "drive W3C over a cliff." 

I respectfully disagree that this is a sufficient substitute for having someone or some small group empowered to ensure that charters advance W3C's mission, would create standards that maintain the conceptual integrity of the web, and are consistent with explicit values.

**Comment by dwsinger** [comment]
Mike, in a member-led consortium, for that is what we are and formally becoming, it's the members' responsibility. We can't duck it, and we shouldn't delegate it either. We elect the TAG and AB to look at these, and if it takes a formal objection, and a community debate, and a formal decision from the Council, that's good. Forcing us to measure questions against values, and not delegating that wrestling, is good for the community.

**Comment by michaelchampion** [comment]
@dwsinger I agree with the "members responsibility" point.  But the FO process is an anachronism unless there is some Director (Directorate, Technical Director, whatever) to appeal to.  The Council punted the most recent "hard" question about DID to the Founding Director (so I hear), and that's not actually in the Process yet.  So I for one have little confidence that the Council mechanism will work for hard strategic issues.

So given that there's no interest in a Technical Director / Directorate, the way forward seems to be to accept that these are member decisions.  But don't bias the process against NO votes by forcing them to be "formal objections" to the institutional memory of a Director that must meet a high conceptual bar ("run off a cliff") and force the NO voter to fight an almost certain-to-lose public battle to overturn the supposed :"consensus" . 

Just make it a secret ballot vote by the AC to approve a charter or not.

**Comment by michaelchampion** [comment]
The current process creates a MASSIVE bias in favor of approving charters.  It takes zero thought and effort to vote yes (and not be publicly accountable if the charter creates problems down the road), but a willingness to go public (and be ready for abuse on social media) , craft a compelling argument, and defend it repeatedly in order to formally object.

**Comment by mnot** [comment]
@dwsinger there are many tools in the governanc toolbox -- resorting to straight voting by the AC is only one option (and it has some serious downsides, as explained). Many successful systems use bodies that are both accountable to voters and invested in finding the right long-term solution (rather than just the most popular/expedient one).

**Comment by frivoal** [comment]
> I was once told by the team that I had "exceeded my quota of formal objections" by voting against too many charters

I find that outrageous. If you filed frivolous objections, they should be dismissed because they're frivolous, even if you only filled one of them. Otherwise, they should be considered seriously, not matter how many you have.

**Comment by frivoal** [comment]
> The Council punted the most recent "hard" question about DID to the Founding Director (so I hear)

No, it did not.

On January 28th, the AB said it was happy to take it, but that given the rest of its pipeline, it expected a delay of a few weeks, and said that if the Team did not find that acceptable, they had the ability to do it themselves under the current Process[1]. The team decided to do it themselves, as they felt it was urgent and could not wait. [2] (The Team/Director rendered their decision on June 30 [3].)

[1] https://lists.w3.org/Archives/Member/w3c-ac-members/2022JanMar/0021.html
[2] https://lists.w3.org/Archives/Member/w3c-ac-members/2022JanMar/0024.html
[3] https://lists.w3.org/Archives/Public/public-did-wg/2022Jun/0002.html

**Comment by jeffjaffe** [comment]
Although I personally was comfortable with closing this issue - seeing that several people do not agree - I think we need to leave it open.

**Comment by chaals** [comment]
"What is in scope for W3C" is a pretty common bone of serious contention. HTML, Verifiable Credentials, Payments, Linked Data / Semantic Web have all been argued over long and hard in relatively recent memory. Meanwhile it seems topics that don't involve a major shift in the work of a "heavyweight" W3C member are often uncontroversial, however tangential they appear, if there are a reasonable number of interested members who want to work on them.

It seems reasonable in this organisation, better than many at dealing with controversies and strongly entrenched positions, to start from a presumption that what a "sufficient" proportion of members think is in scope is reasonable work to do, rather than inviting people to take as a mission, suggesting that it probably isn't. In other words, I think what @michaelchampion describes as

> zero thought and effort to vote yes (and not be publicly accountable if the charter creates problems down the road), but a willingness to go public (and be ready for abuse on social media) , craft a compelling argument, and defend it repeatedly in order to formally object.

seems reasonable to me. In part because I think it's somewhat inaccurate and somewhat hyperbole. Also in part because I believe that a large number of the "philosophical objections" I have seen over the last couple of decades and more are just ill-disguised efforts to tilt the market in one direction or another or to save someone from having to rework something.

The AC review process is far from perfect, and we are not as efficient as if everything were decided by one very active leader with total control over direction. But even TimBL was never that. While I favour W3C have a formal technical director on staff, i don't think that would change the vital role that AC review - and the subsequent discussion - plays in dealing with these questions when they arise.

**Comment by palemieux** [comment]
Building on https://github.com/w3c/w3process/issues/620#issuecomment-1240121831, it is critical, post-Director, to clearly state in the W3C Process what the mission, values and scope of the W3C are, so that folks have a basis for their objection (and their input in general) -- otherwise it will be just one opinion against another.

**Comment by plehegar** [comment]
As a reminder: "The W3C Process Document describes the organizational structure of the W3C and processes, responsibilities and functions that enable W3C to accomplish its mission." The Process isn't the place to define the mission, values, and scope of the W3C. Instead, it should point to where those things are defined. And the mission/scope/goals should come from some combination of AB/TAG/BoD, within their respective scopes and past decisions.

I see #651 and #653 as a better to discuss the issue of how charters are established. The team filters the request for new charters (based on mission/scope/values)in coordination with AB/TAG, and makes decisions that can be objected.

**Comment by palemieux** [comment]
> The Process isn't the place to define the mission, values, and scope of the W3C. Instead, it should point to where those things are defined. And the mission/scope/goals should come from some combination of AB/TAG/BoD, within their respective scopes and past decisions.

Fine with me as long as:

(a) text that defines the mission, values, and scope of the W3C is removed from Process
(b) new documents are created and referenced from the Process

In the meantime, the Process contains text that defines the mission, values, and scope of the W3C

**Comment by palemieux** [comment]
![image](https://user-images.githubusercontent.com/4871350/193114891-21907677-e6c8-4677-bea8-660d7ea81d25.png)

See screenshot above from https://www.w3.org/2021/Process-20211102/

To be sure, my objective is to make sure that there is a stable and complete set of documents that establish the rules to which members are subject to.

**Comment by mnot** [comment]
> The AC review process is far from perfect, and we are not as efficient as if everything were decided by one very active leader with total control over direction. But even TimBL was never that. While I favour W3C have a formal technical director on staff, i don't think that would change the vital role that AC review - and the subsequent discussion - plays in dealing with these questions when they arise.

Those aren't our only choices, luckily. There are a [wide variety](https://communityrule.info/templates/) of governance structures we can choose from --with huge differences in their properties. The most obvious in this situation are:

* A benevolent dictator (TimBL)
* A petition system involving every member (the AC)
* An elected board/body (the TAG? a new thing?)

The politics in each case work in profoundly different ways. 

A benevolent dictator worked for a long time, but I think we all recognise that it has a dependency on finding the right person, which is difficult. It also makes us vulnerable to that person's absence.

A petition system (which is effectively what is currently specified) creates incentives for partiicpants that may not lead to good outcomes. Because so many AC reps are disengaged, I'm concerned that outcomes will be fragmented, factional, and vulnerable to manipulation.

A board, if we get it right, will take ownership of the vision and scope of the organisation, but still gain legitimacy through being elected. They can make the difficult decisions and assure that a strategy is consistently applied, as opposed to the piecemeal approach that's implied by the AC. It can adapt over time as members come and go, but maintain consistency though the overlap.

This is not about replacing the AC as a backstop to assure that members are on board with new work, BTW -- if we had the TAG (for example) inserted into the process, I don't think we'd remove the AC step. It's about whether using the AC as the sole steering function for new work is wise for the future of the Web.

**Comment by plehegar** [comment]
That screenshot doesn't define the mission nor scope of W3C. It certainly touches on values, without trying to define those values either. They are used as a justification for how the Process is being set up since the Process is a reflection of those values. I agree that consensus/quality/fairness being values of W3C needs to be referenced. We can certainly add a link to something like https://github.com/w3c/AB-public/tree/main/Vision (or, better, make sure it's properly integrated on w3.org if that's not the case already).

**Comment by palemieux** [comment]
> That screenshot doesn't define the mission nor scope of W3C. 

_W3C revolves around the standardization of Web technologies_

**Comment by mnot** [comment]
Could the discussion of what / where the mission and scope of the W3C are be split into a separate issue?

**Comment by mnot** [comment]
Tying this into some discussion that happened on a members' call:

- AC review requires a company to take a position, rather than a person. This often pits company-vs-company, which can discourage open discussion about the merits of a particular question. Open participation as individuals rather than company representatives would help here -- e.g., on a W3C-wide feedback list, rather than one restricted to AC reps.
- The judgement of community sentiment in anything less than a FO is opaque. That creates an incentive to FO, even when it may not be necessary.

**Comment by TzviyaSiegman** [comment]
I think @mnot raises 2 really important points. These are both about the fact that we have no good way of communicating within the larger community.

**Comment by michaelchampion** [comment]
Where do we stand in terms of resolving this issue?  I like mnot's framing in https://github.com/w3c/w3process/issues/620#issuecomment-1262993087. 

> A benevolent dictator worked for a long time, but I think we all recognise that it has a dependency on finding the right person, which is difficult. It also makes us vulnerable to that person's absence.
> A petition system (which is effectively what is currently specified) creates incentives for partiicpants that may not lead to good outcomes. Because so many AC reps are disengaged, I'm concerned that outcomes will be fragmented, factional, and vulnerable to manipulation.
> A board, if we get it right, will take ownership of the vision and scope of the organisation, but still gain legitimacy through being elected. They can make the difficult decisions and assure that a strategy is consistently applied, as opposed to the piecemeal approach that's implied by the AC. ...
> This is not about replacing the AC as a backstop to assure that members are on board with new work, BTW -- if we had the TAG (for example) inserted into the process, I don't think we'd remove the AC step. It's about whether using the AC as the sole steering function for new work is wise for the future of the Web.

I'd be comfortable with a resolution along the following lines:
- The Team have the primary role of crafting/reviewing/mentoring charter development to ensure they align with the mission and values of W3C.  BUT they must be more explicitly accountable to the Board for vetting charters against mission/values than they have been in the past.  (To be blunt, it is NOT OK to draft charters to promote a team member's pet theories, as a way to recruit new members who don't share core values, etc.)

- The AB and TAG are explicitly informed about charters in progress and invited to comment, but the charter process isn't blocked on them responding nor do they get an automatic veto.  I like at least the metaphor of  Green / Yellow / Red lights, with the TAG or AB being able to go on record as saying a charter is OK,  worrisome, or dangerous...  and having that input be considered by the team and AC as the charter gets final consideration

- I would prefer that AC charter ballots not be so heavily stacked in favor of approval as they are now.  Dissent on principled or pragmatic grounds should be encouraged even if approval would not "drive W3C over a cliff".  (This the guidance I got from the team when I was an AC rep dissenting against "too many" charters for their taste.) Either dropping the requirement for formal objections  to be made public might help, or (conversely) making both positive and negative votes a matter of public record and future scrutiny might as well.

**Comment by palemieux** [comment]
> I would prefer that AC charter ballots not be so heavily stacked in favor of approval as they are now. 

I disagree.

the W3C should a priori welcome new projects driven by its members, unless:

1. deliverables will necessarily and demonstrably conflict with the mission, scope or process of the W3C
2. there are no member resources to conduct the project
3. the charter does not meet the [contents requirements](https://www.w3.org/2021/Process-20211102/#charter)

Suggestions to charters should of course be encouraged and welcome, but the fact that a member "does not think the project is worthwhile or succeed" or "would do it differently" should not be grounds for not approving the project.

The AB, TAG and board should only get involved if member(s) believe that the charter fails (1), (2) or (3) above.

**Comment by dwsinger** [comment]
> the W3C should a priori welcome new projects driven by its members, unless:
> 
> 1. deliverables will necessarily and demonstrably conflict with the mission, scope or process of the W3C

Values. Charters should be disapproved, or at least be subject to intense scrutiny, if their aim or likely effect is in direct conflict to our values.

**Comment by palemieux** [comment]
> Values. Charters should be disapproved, or at least be subject to intense scrutiny, if their aim or likely effect is in direct conflict to our values.

+1, assuming "values" is defined somewhere.

**Comment by michaelchampion** [comment]
This is one of the less obvious gaps opened up by not having a Director to *implicitly* define and then make hard decisions based on "our values".  Sure, the Values need to be defined in a fairly clear way (that's another set of issues in https://github.com/w3c/AB-public/issues ).  

Just like asking the AB+TAG to step up to some of the Director roles in FO handling, asking them to make implicit -- for the time being -- value judgments the Director once made about charters makes sense to me.

**Comment by chrisn** [comment]
>  It's been clearly established many times that we wanted the Board to be tasked with running the organization, not the technical program.

Not only that "we wanted", but the AC has been given explicit reassurances from the GovTF that this would be the case.

**Comment by michaelchampion** [comment]
>It's been clearly established many times that we wanted the Board to be tasked with running the organization, not the technical program. Yes, there are gray zones in between, but no, I don't think we want to directly involve the Board in chartering.

Perhaps I was unclear.  I'm just proposing (assuming?) the Board *annually* assess how well the Team has done in vetting charters that "fit the mission and values of W3C" as part of its management oversight function. I don't believe anyone is holding the Team accountable for that now.  W3M has seemed more concerned with recruiting members by offering to help create charters that would attract them, the AB has no power, and the Director/SC were more or less checked out. If it's not the Board holding the Team accountable for effectively pursuing the W3C's mission, who is holding them accountable?

>  "the Team decide what the members should work on"

I'm saying the Team should ensure that the work W3C members do UNDER W3C's AUSPICES  fit W3C's mission and values.  And I'd be very comfortable with an alternative way to ensure charters are appropriate for W3C but the alternatives (another Director, a Technical Director, a more rigorous AC approval mechanism that doesn't make Approve the default) got lots of pushback.

> The only way an SDO works is if the members decide what they want to work on;

I don't think the web standards ecosystem needs another OASIS, which is sort of an 'SDO in a box" for groups who want to create a standard under a well-defined process and IPR system but with few requirements to fit into a larger mission or architecture.

**Comment by dwsinger** [comment]
The Board has direct oversight of the CEO and thus indirect oversight of the staff, indeed. We have initiated the practice of having a staffing function review – what staff do we employ, doing what – but I agree we could go further in accountability.

**Comment by tackoman** [comment]
> Turning @mnot's  https://github.com/w3c/w3process/issues/328#issuecomment-1210093948 into a standalone issue: In a W3C without an engaged Director, how do we ensure that chartered work is consistent with W3C's mission and values?  
> 
> Underlying problem: Initially W3C had a Director to make judgments on how to maintain the conceptual integrity of the w3c's standards system, but now that role is essentially vacant. So we need some mechanism for the community  to evaluate such matters at charter time, and encourage any concerns to be explicitly raised and authoritatively resolved during the approval process.
> 
> Symptoms: W3C's recent public controversies tend to happen when a spec gets to Proposed Recommendation and only then are are fundamental questions surfaced about its appropriateness for W3C's mission, culture, implicit values, etc.  The obvious example is EME https://en.wikipedia.org/wiki/Encrypted_Media_Extensions#Criticism .  We can certainly expect future controversies based on privacy and sustainability concerns.  Once W3C is launched as a non-profit corporation without even nominal supervision from academic institutions or Director applying an explicit vision and implicit values to approve work at various maturity stages, it will be even more important to ensure such issues are resolved early.  Rejecting a spec worked on for years for fundamental philosophical reasons will seem like a betrayal of trust and drive members away.  Conversely, not ensuring that W3C's Recommendations align with a coherent mission and set of values will drive away those who engage in W3C hoping to close gaps between the web's "full potential" and current (sometimes ugly) reality. 
> 
> Possible solutions:
> 
> 1. Tweak the current / proposed Director-free process to ensure charters get horizontal review to surface issues and the AC is informed of these results when voting on a charter. That seems to be the intent of #328, but as @mnot notes that is "sweeping some signficant problems under the carpet." 
> 
> 2. @mnot's suggestion: Make the TAG (or possibly TAG+AB) "responsible for new charters (much as in the IETF)" .  But this will add to the already significant burdens of the TAG, and it's not clear that the current TAG members want the role.
> 
> 3. My previous suggestion in #457 and embedded is other issues for a Technical Director(ate) that would be explicitly hired/elected/appointed to be responsible for issues at the intersection of architecture and strategy such as the problem here.  But that proposal has gotten little support, and raises the obvious question of who exactly W3C could entrust with such responsibility and how to find a qualified independent person W3C could afford to hire.  Plus this would require engagement of the  Board of Directors who will have their hands full restructuring the current organization.
> 
> Perhaps there are other possibilities, or an efficient / workable way of combining some of these options to address the problem.
> 
> Also, perhaps this issue must be deferred until there is an authoritative and operational statement of what W3C's mission and values are.  This has been an AB "priority" for some time now, and perhaps will get some cycles once a BoD takes over the job of advising and supervising the team on "business" matters. But I suspect discussions of the *process* for ensuring the values are respected in charters can proceed in parallel with discussions of the *content* of the values.
> 
> 
> 
>

**Comment by nigelmegitt** [comment]
I'd be in favour of closing this issue with no change. The thesis that there's insufficient scrutiny of charters prior to approval has been demonstrated to be false. All the folk who should be able to review charters can and do already do so. 

Any definition of W3C's mission/values/goals that is open enough to be acceptable to all members is by definition not going to be suitable, by itself, to use as a yes/no threshold measure for whether any proposed charter is acceptable: the decision can only end up with members, which is where it already lies.

**Comment by michaelchampion** [comment]
> The thesis that there's insufficient scrutiny of charters prior to approval has been demonstrated to be false. 

Please explain the logic here.  I opened this issue because I believe the team does relatively little scrutiny of charters to ensure they are consistent with W3C's mission and values. That creates a problem:

> W3C's recent public controversies tend to happen when a spec gets to Proposed Recommendation and only then are are fundamental questions surfaced about its appropriateness for W3C's mission, culture, implicit values, etc. The obvious example is EME

So what evidence has come out in this thread refuting that argument?

**Comment by dwsinger** [comment]
> I'd be in favour of closing this issue with no change. The thesis that there's insufficient scrutiny of charters prior to approval has been demonstrated to be false. All the folk who should be able to review charters can and do already do so.

I'm sure I missed that proof, and don't agree. I think it important that AC review can be informed by any AB and TAG review, if possible, for example.

I think we're still at risk of "if you don't object to my charter I won't object to yours" that is a common pattern in standards bodies, if they are not careful.

**Comment by nigelmegitt** [comment]
@dwsinger @michaelchampion to summarise (somewhat selectively):

* https://github.com/w3c/w3process/issues/620#issuecomment-1240121831 in which @jeffjaffe notes that all stakeholders who believe that a Charter doesn't fit the W3C's mission and values can _already_ object.
* https://github.com/w3c/w3process/issues/620#issuecomment-1240818822 in which @plehegar notes that all Charter go through Horizontal Review. (I note that the HR groups include TAG)
* https://github.com/w3c/w3process/issues/620#issuecomment-1240952653 in which @dwsinger notes that we have had Charters for which the AC has had active "in scope of W3C" discussions, i.e. there is evidence that AC already considers scope as part of the review process. And in which @frivoal proposes that better pre-analysis is done of Charters prior to review to support the members in their decision making.
* https://github.com/w3c/w3process/issues/620#issuecomment-1256729603 in which @palemieux notes that there is not adequate definition of mission and values against which to assess proposed Charters' fit with them in a hard yes/no way (that was my reading of the comment).
* https://github.com/w3c/w3process/issues/620#issuecomment-1351474333 in which @chrisn notes that we have explicit assurances that the Board will not get directly involved in defining the work that W3C does.

The best practical proposal I have seen is @frivoal 's suggestion that the team does more to pre-analyse proposed Charters against W3C's mission and scope and provide their analysis to the AC in advance of the AC Review. I don't believe that requires a Process change. And I do believe that it's only an input to the AC Review and that AC has the final say, rather than Team having a pre-filtering role. Of course the Team could discuss their analysis with the people proposing the Charter and encourage them to make changes so that the Charter has a better fit, if that seems appropriate.

@dwsinger has introduced in the most recent comment a new consideration in this issue thread, that political machinations come into play during AC reviews, with members trading off their votes to suit their own needs. That may be the case, but I think a) it's impossible to regulate against, unless you want to remove the AC review of Charters altogether and b) it seems like a separate issue with a much bigger scope than just Charters: it could equally well apply to transition reviews.

**Comment by palemieux** [comment]
> And I do believe that it's only an input to the AC Review and that AC has the final say, rather than Team having a pre-filtering role. 

+1

**Comment by dwsinger** [comment]
I don't disagree that there have been AC discussions about scope and mission, around Charters. I do disagree that what we do now is good enough. It's too easy for Charter approval to become routine, and now that the Director has left, it's by no means clear whose job it is to ask "should we really be doing this?" from a web architecture, scope, mission, or procedural point of view.

I continue to think that the TAG and AB should be (a) enabled to provide comments on charters, so that the AC can consider them in AC review and (b) expected to do that.

So, a case where we get questions about charter statements being problematic arising 'late in the process' (e.g. at a document transition time) should alert us that (b) above failed. We should aim to catch chartering problems, well, in chartering, not later in the process.

**Comment by chaals** [comment]
>  It's too easy for Charter approval to become routine, and now that the Director has left, it's by no means clear whose job it is to ask "should we really be doing this?" from a web architecture, scope, mission, or procedural point of view.

I thought that was the clear purpose behind the AC review of charters. I'm pretty sure objections have been raised on all those grounds to one charter or another over the last couple of decades.

Given the AC is expected to do the review, I don't think the TAG and AB should be expected to "pre-digest" that review for them. That seems likely to make the AC routinely expect the TAG and AB to do the work and go along with what they suggest, as well as raising the workload for the AB and TAG, and complicating the perception of W3C Council resolutions of such objections as are raised.

However, it is reasonable for individuals on the AB or TAG to raise concerns to the AC.

**Comment by michaelchampion** [comment]
>Given the AC is expected to do the review, I don't think the TAG and AB should be expected to "pre-digest" that review for them. 

The AC may be "expected" to review charters, but in the currently open charter ballot only about 3% of the membership has responded at all, and only one saw fit to make a comment (on a completely editorial point).  Skimming the charter reviews for the last year or so, only one got even 10% of the membership responding, most get no more than 5% of the members responding.  

More importantly, most of the substantive responses (suggest changes with or without a formal objection) are either on Process-related points such as the Success Criteria, or are on Mission/Strategy related point ("W3C should not be doing this").  And formal objections now go to a W3C Council, made up of the TAG and AB.  

So, why NOT have the TAG and AB "pre-digest" charters to give up-front feedback on issues are going to eventually give when Formal Objections are resolved?  It seems like a better use of everyone's energy to have issues about mission/strategy/values and the mechanics of the Process resolved early on.

For example, the HTML Working Group that in principle gives W3C's imprimatur to WHATWG living standards that meet W3C's criteria.  (But hasn't done so in a while for various reasons that seem related to the mechanics of the collaboration).  While the AC clearly has the ultimate authority on whether to approve such a charter, there are so many devils in the details, personal interaction issues, and meta-level abstractions involved that SOMEONE clearly needs to drive a top-down strategy and implementation review to make recommendations on whether to try to fix or give up on the collaboration. Is that a Board responsibility?  A management responsibility?  An AB responsibility? Whatever, it seems extremely inefficient to wait until there is a Formal Objection to a charter or Proposed Recommendation to do the necessary research and consensus-building, then make a strategic decision, then make sure it is implemented in the Process and other foundational documents or policies.

So, "pre-digesting" this kind of stuff is EXACTLY what we need given that the AC reviews don't get the participation or engagement needed to create informed, authoritative decisions.

**Comment by dwsinger** [comment]
The AC is a large-enough group for the problem to be "[someone](https://www.salesproinsider.com/who-is-responsible-everybody-somebody-anybody-or-nobody/) should/will do it" to be common. There needs to be some kind of function to raise concerns to the AC so that when they are voting, they don't just say "another of these, rubber-stamp". 

(Maybe we should send occasional 'test charters' for approval that should obviously be disapproved, to see whether people are paying attention!)

**Comment by michaelchampion** [comment]
> (Maybe we should send occasional 'test charters' for approval that should obviously be disapproved, 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1420798/ 
> Peer review might also be useful for detecting errors or fraud. At the BMJ we did several studies where we inserted major errors into papers that we then sent to many reviewers.[3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1420798/#ref3),[4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1420798/#ref4) Nobody ever spotted all of the errors. Some reviewers did not spot any, and most reviewers spotted only about a quarter. Peer review sometimes picks up fraud by chance, but generally it is not a reliable method for detecting fraud because it works on trust.

**Comment by michaelchampion** [comment]
@dwsinger Thanks for the reminder about the [someone](https://www.salesproinsider.com/who-is-responsible-everybody-somebody-anybody-or-nobody/) poem ;-) 

>  It ended up that Everybody, blamed Somebody, when Nobody did, what Anybody could have done.

**Comment by mnot** [comment]
Exactly. Our current design effectively relies on someone to FO so that the Council can make a reasoned decision about what we can / should / will do.  That makes the process needlessly adversarial and expensive if we want good (for some definition of good) charters. 

Of course, if we want to wave through any charter that gets support from a few members and lacks anyone with enough energy to object, it's a great design.

**Comment by cwilso** [comment]
I've filed a related issue for the AB to discuss.  https://github.com/w3c/AB-memberonly/issues/149

**Comment by michaelchampion** [comment]
Closing in deference to https://github.com/w3c/AB-memberonly/issues/149
